Test/Attestation/AppleAppAttest.cs

```diff
@@ -0,0 +1,152 @@
+        var cbor = Convert.FromBase64String(b64);
+        _attestationObject = (CborMap)CborObject.Decode(cbor);
+        _aaguid = new("61707061-7474-6573-7464-6576656c6f70");
+
+        //_attestationObject = new ParsedAttestationObject
+        //(
+        //    fmt: (string)json["fmt"],
+        //    attStmt: (CborMap)json["attStmt"],
```

Comment:
What about the `receipt` which is part of attStmt?
This will be needed if the counter field needs to be verified later:
https://developer.apple.com/documentation/devicecheck/assessing_fraud_risk?language=objc

Reply:
I don't know what people do with the `receipt` field and it's not in the WebAuthn spec so I was just ignoring that for now.  The `counter` is part of the `authData` field and is handled just like any other attestation format.